### PR TITLE
add a minimal example of using 2 HC595 chained to scan columns and 1 …

### PR DESCRIPTION
**Pull request by alinelena**
…HC589 to read rows, rgb, oled, unicode are also shown

### CODE REVIEW

File: qmk_porting/keyboards/m8rev1/keymaps/default/rules.mk
Comment (Huckies) [inline qmk_porting/keyboards/m8rev1/keymaps/default/rules.mk:1]: this file is not necessary

File: qmk_porting/keyboards/m8rev1/m8rev1.c [hunk 1]
```diff
@@ -0,0 +1,209 @@
+// Copyright 2023 alin elena <https://github.com/alinelena> <https://gitlab.com/drFaustroll>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+#include "m8rev1.h"
```
Comment (Huckies) [inline qmk_porting/keyboards/m8rev1/m8rev1.c]: `QMK_KEYBOARD_H` is set to `m8rev1` by the building system, see[https://github.com/O-H-M2/qmk_port_ch582/blob/via/qmk_porting/qmk_porting.cmake#L72](url)

File: qmk_porting/keyboards/m8rev1/qmk_config.h
Comment (Huckies) [inline qmk_porting/keyboards/m8rev1/qmk_config.h:1]: please reference the latest config file
e.g. [https://github.com/O-H-M2/qmk_port_ch582/blob/via/qmk_porting/keyboards/m2wired/qmk_config.h](url)

File: qmk_porting/keyboards/m8rev1/keymaps/default/rules.mk
Comment (alinelena) [reply qmk_porting/keyboards/m8rev1/keymaps/default/rules.mk:1]: indeed left over

**Review by Huckies — APPROVED** [review]

**Pull request merged**